### DISCUSSION TITLE
creating website

### DISCUSSION THREAD

**Discussion by GOLDENWINDREQUIEM** [community/community#136350] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

So I have trouble creating github pages website. I have a webpage and index and it is uploaded to github. It is not working. goldenwindrequiem.

**Comment by KyNaOo** [comment]
Hey, here is the documentation about gitpage and how to do it : 
https://pages.github.com/
once repository created you can access to your website with an URL that should look like this "https://username.github.io", if the documentation isn't clear for your. 
Here is an video explaining it step by step :
https://www.youtube.com/watch?v=BT4WzyT2g8k

have a great day !

**Comment by trandangtrungduc** [comment]
You can follow this template for your website with Jekyll:
1. How to install, step, deploy,... at [here](https://github.com/alshedivat/al-folio/blob/master/INSTALL.md#deployment).
2. This blogs also talk about [How to Create a Static Website with Jekyll](https://www.taniarascia.com/make-a-static-website-with-jekyll/)
3. Or [this video](https://www.youtube.com/watch?v=g6AJ9qPPoyc&t=435s) I followed to make my website.

Good luck
